lms/templates/certificates/_accomplishment-banner.html

```diff
@@ -0,0 +1,35 @@
+<div class="wrapper-banner wrapper-banner-user">
```

Comment:
NOTE: banner displayed to a logged in user who's viewing their own accomplishment.